src/gui/transferlistmodel.cpp

```diff
@@ -505,6 +517,10 @@ QVariant TransferListModel::internalValue(const BitTorrent::Torrent *torrent, co
         return torrent->distributedCopies();
     case TR_TOTAL_SIZE:
         return torrent->totalSize();
+    case TR_INFOHASH_V1:
+        return torrent->infoHash().v1();
```

Comment:
To fix the CI error:
```suggestion
        return QVariant::fromValue(torrent->infoHash().v1());
```

Reply:
Okay, logical, the function needs to return `QVariant` not `SHAXHash`.